### ISSUE TITLE
InceptionV1/Conv2d_1a_7x7/BatchNorm/beta not found in checkpoint

### ISSUE THREAD

**Issue by Junhyuk** [hukim1112/slim#1]
스터디 중에  runner 실행하다가 문제가 있어서 남깁니다 

아래 다음에서 Key InceptionV1/Conv2d_1a_7x7/BatchNorm/beta not found in checkpoint
이런 에러가 발생합니다. my_cnn 을 바꿔 줘야하는지요 ?

 Fine-tune the model on a different set of labels.
